src/pages/resource/ContributorsPanel.tsx

```diff
@@ -1,27 +1,45 @@
-import React, { useReducer } from 'react';
-import { useTranslation } from 'react-i18next';
+import React, { useReducer, useState } from 'react';
 import { useSelector } from 'react-redux';
 
 import mockContributors from '../../utils/testfiles/contributors.json';
+import { USE_MOCK_DATA } from '../../utils/constants';
 
 import TabPanel from '../../components/TabPanel/TabPanel';
 import { contributorReducer } from '../../redux/reducers/contributorReducer';
 import { RootStore } from '../../redux/reducers/rootReducer';
-import Contributor from './contributors/Contributor';
-import ContributorLabel from './contributors/ContributorLabel';
-import ContributorValidator from './contributors/ContributorValidator';
-import StyledContributor from './contributors/StyledComponents';
+import Contributors from './contributors/Contributors';
+import { addContributor } from '../../redux/actions/contributorActions';
```

Comment:
Jeg syns det er litt rart at vi nå har ting som ikke ligger i redux i redux folder. Kunne gjerne tenkt meg at vi bare kutter ut redux her, og heller flytter `actions` og `reducers` ett nivå opp, eller at vi bare flytter actions/reducers som ikke er i redux ut fra redux folder.
Hva tenker dere? Denne jobben vil uansett bli en seperat PR

Reply:
tja, det er jo redux-funksjonalitet, men det ligger ikke i _storen_, nei. Vi kan flytte actions og reducers ut av redux-mappa (sånn som vi hadde det før), eller så kan vi ha en mappe kalt contributors/local med contributorActions og contributorReducer evt da? men vi kan uansett ta en diskusjon på dette 👍

Reply:
er det redux-funksjonalitet? Det importeres jo fra 'react', men det er egentlig en "kopi" av hvordan redux har gjort ting tidligere (hvis jeg har skjønt useReducer riktig)

Reply:
joda, det er sant at react har speilet redux på useReducer, men det er fortsatt Redux hvis vi snakker om konseptet Redux, og ikke lokasjonen. For meg er Redux en tankemåte (actions, reducers, store og dispatch) og det er jo det som brukes når vi bruker useReducer. Men er åpen for å legge lokale reducere og tilhørende actions en annen plass enn i redux-mappa hvis det er forvirrende

Reply:
har ikke problemer med at det ligger under redux-mappa, men mener vi likevel bør splitte det ut i f.ex underkataloger, siden de ikke hører til i root-reducer, samt om antallet filer blir stort